SPECS/glusterfs/glusterfs.spec

```diff
@@ -791,7 +1116,7 @@ exit 0
 ##
 %pre
 getent group gluster > /dev/null || groupadd -r gluster
-getent passwd gluster > /dev/null || useradd -r -g gluster -d %{_rundir}/gluster -s %{_sbindir}/nologin -c "GlusterFS daemons" gluster
+getent passwd gluster > /dev/null || useradd -r -g gluster -d %{_rundir}/gluster -s /sbin/nologin -c "GlusterFS daemons" gluster
```

Comment:
Please leave the path macros instead of the hard-coded strings.

Reply:
all the changes here are upstream changes.  my goal was to align to upstream except when the mariner variable is used to define behavior specific to mariner.  i can make this change, but it is going to make the next upgrade harder.

Reply:
modified per suggestion